frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/runLogs/DownloadDropdown.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import {
+  Dropdown,
+  DropdownItem,
+  DropdownToggle,
+  KebabToggle,
+} from '@patternfly/react-core/deprecated';
+import { Truncate } from '@patternfly/react-core';
+
+type DownloadDropdownProps = {
+  onDownload: () => void;
+  onDownloadAll: () => void;
+  isSmallScreen: boolean;
+  isSingleStepLogsEmpty: boolean;
+};
+
+const DownloadDropdown: React.FC<DownloadDropdownProps> = ({
+  onDownload,
+  onDownloadAll,
+  isSmallScreen,
+  isSingleStepLogsEmpty,
+}) => {
+  const [isDownloadDropdownOpen, setIsDownloadDropdownOpen] = React.useState(false);
+
+  return isSmallScreen ? (
+    <Dropdown
+      toggle={<KebabToggle onToggle={() => setIsDownloadDropdownOpen(!isDownloadDropdownOpen)} />}
+      isOpen={isDownloadDropdownOpen}
+      isPlain={isSmallScreen}
+      dropdownItems={[
+        <DropdownItem
+          isDisabled={isSingleStepLogsEmpty}
+          key="current-container-logs"
+          onClick={onDownload}
+        >
+          <Truncate content={'Download current step log'} />
+        </DropdownItem>,
+        <DropdownItem key="all-container-logs" onClick={onDownloadAll}>
+          <Truncate content={'Download all step logs'} />
```

Comment:
This will cause the linter to fail when this is brought into `main`.
```suggestion
          <Truncate content="Download all step logs" />
```